service/controller/clusterapi/v17/key/common.go

```diff
@@ -0,0 +1,9 @@
+package key
+
+const (
+	LabelOperatorVersion = "aws-operator.giantswarm.io/version"
```

Comment:
This should probably be in `pkg/label`.

Reply:
Good shout. I also fixed the operator name.

Reply:
I really like that we have `pkg/label` in `cluster-operator` but I'm also thinking we should have a library for labels and annotations. Since we are using them more and there are dependencies between operators. 

Something like `giantswarm/metadata`. I think this was discussed before but I'm not sure what the decision was?

Reply:
Not sure. I like the label package in the projects. That would be a good step. We always have to mix that somehow and if a separate repo for 10 common labels makes so much sense, I do not know. When you think it is the right thing, let's bring it up and maybe just drive the change.

Reply:
Yes, I'm also a fan of the label package. I think adding it to operators that's don't have it is a good first step.

We can always extract to a repo later. I'd like to do that, but I don't have enough bandwidth at the moment.